bazel/external/quiche.BUILD

```diff
@@ -50,7 +50,10 @@ genrule(
     visibility = ["//visibility:private"],
 )
 
-quiche_copt = ["-Wno-unused-parameter"]
+quiche_copt = [
```

Comment:
@htuch Are these quiche-only options going to be leaked to envoy code once envoy depends on quiche?

Reply:
You can check this by compiling with the bazel option that echoes commands.  I think it might be --show_subcommands or similar.

Reply:
In integration code we can use `#pragma GCC diagnostic ignored` to suppress the warning as I commented above. This is what grpc does.

Reply:
Great. In general, when something is non-intuitive in the code in the PR, and I ask about it, what I'm looking for is to add the comment to the code, not just the github review.

Reply:
added more comment.

Reply:
thanks.